src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -282,7 +287,8 @@ function VAFacilityPageV2({
                 loadingParents ||
                 loadingFacilities ||
                 loadingEligibility ||
-                (facilities?.length === 1 && !canScheduleAtChosenFacility)
```

Comment:
The regular `facilities` property from Redux now includes supported and unsupported facilities